README.md

```diff
@@ -66,6 +66,8 @@ pip install -U pip wheel setuptools
 pip install concrete-python
 ```
 
+**Note:** Not all versions are available in PyPI. If you are looking for a version you can't find on PyPI (or even nightly releases), you can add `--extra-index-url https://pypi.zama.ai` to install from our own package index.
```

Comment:
Revised:
Not all versions are available on PyPI. If you need a version that is not on PyPI (including nightly releases), you can install it from our package index by adding `--extra-index-url https://pypi.zama.ai`.